### ISSUE TITLE
Use different icon for "Global" settings

### ISSUE THREAD

**Issue by sleupold** [dnnsoftware/Dnn.AdminExperience#1042]
## Description of bug
Any settings, which do not affect current site only in a multi tenant DNN installation, gets marked with a special icon, currently a globe icon. However, in a multilingual site, this could be misinterpreted to affect special regions or languages only. I suggest to use a different icon, which is not connotated to regions or languages

## Steps to reproduce
List the steps to reproduce the behavior:
Login as Admin or Superuser, enter e.g. Server > Server Settings > SMTP Server.

## Current result
Note the icon to the left of most settings, when in host mode. This looks like a globe.

## Expected result
use an icon, which cannot be misunderstood to be language or region specific.

## Affected version

* [x] 9.2.2
* [x] 9.2.1
* [x] 9.2
* [x] 9.1.1
* [x] 9.1
* [x] 9.0

## Affected browser

* [x] Chrome
* [x] Firefox
* [x] Safari
* [x] Internet Explorer
* [x] Edge

**Comment by mitchelsellers** [comment]
I like the idea of a "Server" icon or similar.  This WILL require extensive edits though to verbiage & documentation.

**Comment by valadas** [comment]
Yes, the change in the PersonaBar modules would be a quick thing, 1 or 2 common components to swtith the icon on. But for the documentation team, they would have to take new screenshots. I think it is worth it though, that globe never made sense to me as a representation of "global" for me it really means "localizable" or something along those lines. We could actually move it to fields that are localizable, or maybe a neutral flag on localizable fields (page name, description, logo, etc)

**Comment by sleupold** [comment]
global IMO is associated with languages, therefore a globe would be more suitable for language independent options.
